docs/modules/ROOT/pages/operations.adoc

```diff
@@ -292,22 +292,23 @@ Possible values: `auto`, `normal`, `outline`, and `dark`.
 
 | `browserSyncDebugServer`
 | `null`
-| TODO
+| Internal setting for debugging browser sync.
 | yes
 
 | `maxRows`
 | `1000`
-| TODO: The maximum rows to render in the "Row" result view.
+| The maximum rows to render in the *Table* and *Text* result views.
 | yes
 
 | `maxFieldItems`
 | `500`
-| The maximum number of items per field.
+| Limit the length of the returned lists.
+For example, if a node has 1000 labels, the list will be cut off to the value set in `maxFieldItems`.
 | no
 
 | `shouldReportUdc`
 | `true`
-| TODO
+| Report less user metrics.
```

Comment:
This is what the setting is supposed to do, but it doesn't work

Reply:
yeah, but we can't say in the docs - sorry, it doesn't work.

Reply:
In other words, these are descriptions of the settings. If they don't work, we should fix them, I guess 😄

Reply:
OK! Not sure if we're going to fix or remove them though

Reply:
What is `Udc` ?

Reply:
user data collection, but the setting is like i said not currently wired to anything. so it might as well stand for anything